api/python/provisioner/commands/__init__.py

```diff
@@ -1207,6 +1207,15 @@ def _passwordless_ssh():
             targets=targets
         )
 
+        # Ensure rsync is installed, if not already
+        StateFunExecuter.execute(
```

Comment:
Common package installation should be covered in:
`cortx-prvsnr/srv/components/system/install.sls`
Which is already present. Although it's idempotent, it should not be repeated unless there's a good reason for doing so.

Reply:
Agree. That command should base on assumption: user is created on field, it means the sw stack is fully deployed and all our necessary states are applied. Otherwise you would need to install here all packages that provides the tools listed for sudo.